Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 license = "MIT OR Apache-2.0 OR CC0-1.0"
 
 [dependencies]
-bevy = { version = "0.14", features = ["wayland"] }
```

Comment:
Why remove `wayland` support? Also, `x11` is a default feature, so no need to specify it explicitly.

Reply:
It's a temporary change to prove that this PR works! I'm reverting 7188e2c623e4b4fc97cd9abad3d55c3ae364544d after the workflow finishes :sweat_smile:

Reply:
Right that makes total sense :)